lib/miq_automation_engine/service_models/miq_ae_service_authentication_rhsm.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceAuthenticationRhsm < MiqAeServiceAuthentication
```

Comment:
I find it strange that only 1 of the many new authentications are exposed to automate... I think if any of the other subclasses come back to automate it will blow up... @gmcculloug ?